install_config/configuring_aws.adoc

```diff
@@ -313,3 +313,5 @@ Balancing (ELB) will not be able to create a load balancer.
 ====
 
 After tagging any resources, restart the master services on the master and the node service on all nodes. See the xref:aws-applying-configuration-changes[Applying Configuration Section].
+
+include::install_config/topics/red_hat_openshift_container_storage.adoc[leveloffset=+2]
```

Comment:
You need to add this line somewhere before line 317:

 :context: configuring-for-aws